client/src/components/papi.js

```diff
@@ -11,12 +11,23 @@ export default function Papi({ selectedPatientID }) {
         calculatedValue: ""
     });
     const [placeholderText, setPlaceholderText] = useState("");
+    const [error, setError] = useState(null);
 
     useEffect(() => {
+        if (ra === 0) {
```

Comment:
Comparison needs to use `==` in this instance. `ra` is a String but `0` is a Number.